seed/views/v3/users.py

```diff
@@ -111,91 +112,18 @@ class ListUsersResponseSerializer(serializers.Serializer):
     users = EmailAndIDSerializer(many=True)
 
 
-class UserSchema(AutoSchemaHelper):
-    def __init__(self, *args):
-        super().__init__(*args)
-
-        self.manual_fields = {
-            ('GET', 'list'): [],
-            ('POST', 'create'): [
-                self.org_id_field(),
-                self.body_field(
-                    name='New User Fields',
-                    required=True,
-                    description="An object containing meta data for a new user: \n"
-                                "- Required - first_name, last_name, email \n"
-                                "- Optional - role ['viewer'(default), 'member', or 'owner']",
-                    params_to_formats={
-                        'first_name': 'string',
-                        'last_name': 'string',
-                        'role': 'string',
-                        'email': 'string'
-                    })
-            ],
-            ('GET', 'current_user_id'): [],
-            ('GET', 'retrieve'): [self.path_id_field(description="users PK ID")],
-            ('PUT', 'update'): [
-                self.path_id_field(description="Updated Users PK ID"),
-                self.body_field(
-                    name='Updated user fields',
-                    required=True,
-                    description="An object containing meta data for a updated user: \n"
-                                "- Required - first_name, last_name, email",
-                    params_to_formats={
-                        'first_name': 'string',
-                        'last_name': 'string',
-                        'email': 'string'
-                    }
-                )],
-            ('PUT', 'default_organization'): [self.org_id_field(),
-                                              self.path_id_field(description="Updated Users PK ID")],
-            ('POST', 'is_authorized'): [
-                self.org_id_field(),
-                self.path_id_field(description="Users PK ID"),
-                self.body_field(
-                    name='Actions',
```

Comment:
I'm not sure. It looks like there's not really a way to easily "name" the body using the swagger decorator ([see options here](https://drf-yasg.readthedocs.io/en/stable/drf_yasg.html#drf_yasg.utils.swagger_auto_schema)) and I guess I wouldn't really consider it a problem since you can add a description to the body

Reply:
Agree - just wanted to make sure we weren't losing anything by losing the `name` attribute 👍